next/components/ui/VisibilityIntersection/useIntersectionObserver.ts

```diff
@@ -6,26 +6,21 @@ interface HookResult {
 
 export const useIntersectionObserver = (
```

Comment:
There is probablu similar hook in `usehoooks-ts` library. Please use that one if possible.
https://usehooks-ts.com/react-hook/use-intersection-observer